### PR TITLE
Tag component

### PR DESCRIPTION
**Pull request by mbutsykin**
# ✨ Pull Request

### 💁 Description
[Tag component](https://docs.google.com/document/d/1Q7HL6w0b2bN-C5ViWKECyfcTCBcuu6taxO7F6d37hKo/edit) implementation

### 👀  PR is ready for review 

- [X] Ready for review

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_585/

**Review by jonathanadler — COMMENTED** [review]
Very nice work!

I've left some small comments, and one a bit bigger 😬 .
Which is to use class names instead of stylable states (like we used to do before stylable 😄 ).

In addition I'm so sorry, but it seems I forgot to define the `TextColor` variable,  in the spec doc. 
It's defined [here](https://zeroheight.com/7sjjzhgo2/p/05a744-05-selections/t/057895) and should be added (I believe you'll also need this in your app, no?)

Thanks!

File: .storybook/stylable-metadata-plugin.js [hunk 1]
```diff
@@ -165,37 +165,39 @@ const getOverridableVars = ast => {
...
+        ...data.metadata,
+        stylable: {
+          overridableVars,
+        }
+      }
+    }
+  }
+
+  return data;
 };
```
Comment (jonathanadler) [inline .storybook/stylable-metadata-plugin.js:203]: It seems that the pull rebase didn't work well 🤔 
This file and the `applitools.config.js` file shouldn't be here

**Review by jonathanadler — COMMENTED** [review]
👏 👏 👏 

Awesome work!

I'll ask our designer to give the design review

Thanks!

File: .storybook/stylable-metadata-plugin.js (hunk 1 above)
Comment (mbutsykin) [reply .storybook/stylable-metadata-plugin.js:203]: indeed. sorry)

**Pull request merged**